src/crc32c.c

```diff
@@ -324,6 +324,9 @@ CRC_TARGET static uint32_t crc32c_armv8(uint32_t crc, const char *buf, size_t le
 }
 
 // HW feature detection
+#ifndef HWCAP_CRC32
```

Comment:
Is this not defined by the auxv header? (That was the whole point....)

Reply:
No, the header doesn't define any features. https://github.com/freebsd/freebsd/blob/master/sys/sys/auxv.h

Reply:
Is this defined anywhere then? It'll also be better to condition this on freebsd, since I don't think these values are universal (it does seem that the freebsd value is the same as the linux one in this case)

Reply:
It appears not: https://github.com/freebsd/freebsd/search?q=hwcap_crc32&unscoped_q=hwcap_crc32

Reply:
No, there are no `HWCAP` defines at all.